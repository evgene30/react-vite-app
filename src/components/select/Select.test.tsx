import React from 'react';
import { render, screen } from '@testing-library/react';
import { expect } from '@jest/globals';
import { SelectLabels } from '.';

describe('SelectLabels', () => {
  render(<SelectLabels />);
  test('snapshot SelectLabels component', () => {
    const defaultValue = '+ Add Filter';
    const selectedValue = screen.getByText(defaultValue);

    expect(selectedValue).toMatchSnapshot();
  });
});
