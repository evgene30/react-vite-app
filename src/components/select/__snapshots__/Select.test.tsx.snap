// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectLabels snapshot SelectLabels component 1`] = `
<label
  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined css-176f7eb-MuiFormLabel-root-MuiInputLabel-root"
  data-shrink="false"
>
  + Add Filter
</label>
`;

exports[`renders SelectLabels component 1`] = `
<label
  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined css-176f7eb-MuiFormLabel-root-MuiInputLabel-root"
  data-shrink="false"
>
  + Add Filter
</label>
`;

exports[`snapshot SelectLabels component 1`] = `
<label
  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined css-176f7eb-MuiFormLabel-root-MuiInputLabel-root"
  data-shrink="false"
>
  + Add Filter
</label>
`;
